pmd-dart/src/test/resources/net/sourceforge/pmd/cpd/testdata/escape_sequences.dart

```diff
@@ -1,3 +1,3 @@
 var newline = '\n';
-var dollar = '$';
+var dollar = '$newLine';
```

Comment:
So `'$'` is not a valid literal, correct?

Reply:
Indeed, that's my understanding. You can either use the dollar sign for string interpolation, or escape it with a backslash character. The previous grammar was a bit too lenient in this regard.

Reply:
Ok thanks for explaining